.github/workflows/format-code.yml

```diff
@@ -18,21 +18,25 @@ jobs:
         with:
           app_id: ${{ secrets.BOT_APP_ID }}
           private_key: ${{ secrets.BOT_PRIVATE_KEY }}
-          
+
+      - run: npm install prettier-plugin-jsdoc prettier
+
       - name: Prettify code (ScratchAddons)
         if: steps.generate-token.outcome == 'success'
         uses: creyD/prettier_action@v3.1
         with:
           prettier_options: --write .
           commit_message: Format code
+          prettier_plugins: "prettier-plugin-jsdoc"
```

Comment:
This won't work since prettier_plugins only work with libs that starts with `@prettier/`. I need to do some changes with this. Stand by.

Reply:
I'm not sure if we want to prettify JSDoc in the first place tbh